doc/quick-start-user-guide.md

```diff
@@ -209,3 +209,54 @@ Verify that `glusterfsd` process is running on both nodes.
 * Issues with 2 node clusters
   * Restarting glusterd2 does not restore the cluster
   * Peer remove doesn't work
+
+### Solutions to common issues
+
+  >  Note: These are hacks or temporary solutions to the problems. Only use these solutions only if the issues are faced during re-setup.
+
+* If glusterd service fails with error: "failed to start embedded store"
+
+  Sample output:
+  ```log
+  ERRO[2018-06-04 06:05:41.020313] failed to start embedded store                error="dial tcp {IP}: connect: connection refused" source="[embed.go:36:store.newEmbedStore]"
+  ```
+  To solve this you will be required to clean up glusterd2 [working directory](https://github.com/gluster/glusterd2/blob/master/doc/quick-start-user-guide.md#running-glusterd2).
+  The path to default directory used by glusterd2 is "/var/lib/glusterd2/", if using custom config file then please provide working directory path instead of "/var/lib/glusterd2/"
+
+  ```sh
+  # rm /var/lib/glusterd/2*
```

Comment:
Is this path correct? Above you write "/var/lib/glusterd2/" but here the last slash and the 2 are transposed.